### ISSUE TITLE
Missing required tags

### ISSUE THREAD

**Issue by Madis0** [Earthcomputer/multiconnect#59]
A crash on 1.16 client and 1.15.2-compatible (ViaVersion) server.
Obtained via launcher log using Not Enough Crashes.
Fabric Loader 0.8.8 build 202, API 0.13.1 build 370.

P.S. For some reason native 1.15.2 servers crash without a crashlog or launcher log error...

```
Error executing task on Client
java.lang.IllegalStateException: Missing required tags: blocks:minecraft:bamboo_plantable_on,blocks:minecraft:beds,blocks:minecraft:bee_growables,blocks:minecraft:beehives,blocks:minecraft:coral_plants,blocks:minecraft:crops,blocks:minecraft:dragon_immune,blocks:minecraft:flowers,blocks:minecraft:portals,blocks:minecraft:shulker_boxes,blocks:minecraft:small_flowers,blocks:minecraft:tall_flowers,blocks:minecraft:trapdoors,blocks:minecraft:underwater_bonemeals,blocks:minecraft:wither_immune,blocks:minecraft:wooden_fences,blocks:minecraft:wooden_trapdoors,entity_types:minecraft:arrows,entity_types:minecraft:beehive_inhabitors,entity_types:minecraft:raiders,entity_types:minecraft:skeletons,items:minecraft:beds,items:minecraft:coals,items:minecraft:fences,items:minecraft:flowers,items:minecraft:lectern_books,items:minecraft:music_discs,items:minecraft:small_flowers,items:minecraft:tall_flowers,items:minecraft:trapdoors,items:minecraft:walls,items:minecraft:wooden_fences,items:minecraft:wooden_trapdoors
	at Not Enough Crashes deobfuscated stack trace.(1.16+build.1)
	at net.minecraft.client.network.ClientPlayNetworkHandler.checkRequiredTags(ClientPlayNetworkHandler:2503)
	at net.minecraft.client.network.ClientPlayNetworkHandler.handler$zjd000$onOnSynchronizeTags(ClientPlayNetworkHandler:2476)
	at net.minecraft.client.network.ClientPlayNetworkHandler.onSynchronizeTags(ClientPlayNetworkHandler:1527)
	at net.minecraft.network.packet.s2c.play.SynchronizeTagsS2CPacket.apply(SynchronizeTagsS2CPacket:31)
	at net.minecraft.network.packet.s2c.play.SynchronizeTagsS2CPacket.apply(SynchronizeTagsS2CPacket:9)
	at net.minecraft.network.NetworkThreadUtils.method_11072(NetworkThreadUtils:21)
	at net.minecraft.util.thread.ThreadExecutor.executeTask(ThreadExecutor:144)
	at net.minecraft.util.thread.ReentrantThreadExecutor.executeTask(ReentrantThreadExecutor:23)
	at net.minecraft.util.thread.ThreadExecutor.runTask(ThreadExecutor:118)
	at net.minecraft.util.thread.ThreadExecutor.runTasks(ThreadExecutor:103)
	at net.minecraft.client.MinecraftClient.render(MinecraftClient:988)
	at net.minecraft.client.MinecraftClient.run(MinecraftClient:654)
	at net.minecraft.client.main.Main.main(Main:215)
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:564)
	at net.fabricmc.loader.game.MinecraftGameProvider.launch(MinecraftGameProvider.java:192)
	at net.fabricmc.loader.launch.knot.Knot.init(Knot.java:140)
	at net.fabricmc.loader.launch.knot.KnotClient.main(KnotClient.java:26)
```

**Comment by Earthcomputer** [comment]
What version are you trying to connect to the server on, and what other mods do you have installed on the client?

**Comment by Madis0** [comment]
Hmm, I was able to narrow the crash down to EasierCrafting, which does seem like would be incompatible with this mod (1.16 recipes...)

**Comment by Earthcomputer** [comment]
I believe you are experiencing two separate issues: one mod incompatibility, and one issue with ViaVersion which I have reported on their issue tracked. EasierCrafting doesn't immediately sound like it should be incompatible, especially since it's the server that sends the client all the recipes. Mind opening a separate issue for that and send the crash log for that other crash?

**Comment by Madis0** [comment]
Well, I don't have any crash logs and I'm not sure why.

**Comment by Earthcomputer** [comment]
Does it cause the client to close? If there's no crash log, maybe try sending the game log instead?

**Comment by Earthcomputer** [comment]
ViaVersion fixed the tags issue in their 1.16.1 release. Can you confirm if it's still an issue on a 1.16.1 client?

**Comment by Madis0** [comment]
Doesn't seem to crash anymore.
